docs/articles/mirroring.md

```diff
@@ -1,13 +1,16 @@
 # OCI Registry Mirroring With zot
 
 > :point_right: A `zot` registry can mirror one or more upstream OCI registries, including popular cloud registries such as Docker Hub and Google Container Registry (gcr.io).
->
 
 A key use case for zot is to act as a mirror for upstream registries. If an upstream registry is OCI distribution-spec conformant for pulling images, you can use zot's `sync` feature to implement a downstream mirror, synchronizing OCI images and corresponding artifacts.
 
-As with git, wherein every clone is a full repository, you can configure a local zot instance to be a full OCI mirror registry. This allows for a fully distributed disconnected container image build pipeline.
+## Mirroring modes
 
-Synchronization between upstream and downstream registries can be implemented by periodic polling of the upstream registry or synchronization can occur on demand, when a user pulls an image from the downstream registry.
+For mirroring an upstream registry, two common use cases are a full mirror or a pull-through cache registry.
+
+The full mirror mode allows for a fully distributed disconnected container image build pipeline.  As with git, wherein every clone is a full repository, you can configure your local zot instance to be a full OCI mirror registry. For this mode, configure zot for synchronization by periodic polling, not on-demand. Zot copies and caches a full copy of every image on the upstream registry, updating the cache whenever polling discovers a change in content or image version at the upstream registry. 
+
+Pull-through cache mode can conserve memory in the mirror registry. For this mode, configure zot for on-demand synchronization. When an image is first requested from the local zot registry, the image is downloaded from the upstream registry and cached. Subsequent requests for the same image are served from zot's cache. Images that have not been requested are not downloaded. If a polling interval is also configured, zot periodically polls the upstream registry for changes, updating any cached images if changes are detected. 
```

Comment:
"Pull-through cache mode can conserve memory in the mirror registry."
^ remove this.

Reply:
change "from the upstream registry and cached" to 
"from the upstream registry and cached in local storage"